classification/view_all_classified_ics.m

```diff
@@ -0,0 +1,58 @@
+function view_all_classified_ics(class_file,ica_mat,label_option)
+% View ICs on top of cell map, where color of outline indicates
+%   cell (green) or not (cell)
+%   
+%   class_file = classification text file
+%   ica_mat = ica mat file
+%   label_option = 'y' if you want IC number labels; 'n' if not
+%
+% 2015 02 19 Fori Wang
+
+figure;
+
+% load ICs
+load(ica_mat);
+
+% Pull out classification data
+fid = fopen(class_file);
+results = textscan(fid,'%s %s %s %s');
```

Comment:
The classification file a comma-separated file currently takes the form:
```
...
37, cell
38, cell
39, not a cell
40, cell
...
```

So, it's not correct to scan this file by `textscan(fid,'%s %s %s %s')`, which means to look for four columns of strings per line.

What you should do is to use the existing function [`load_classification(...)`](https://github.com/inanhkn/analysis/blob/master/classification/load_classification.m) for reading the classification file.